airflow/cli/commands/remote_commands/config_command.py

```diff
@@ -227,6 +227,21 @@ def message(self) -> str:
         suggestion="Remove TaskContextLogger: Replaced by the Log table for better handling of task log "
         "messages outside the execution context.",
     ),
+    ConfigChange(
+        config=ConfigParameter("logging", "log_filename_template"),
```

Comment:
Will this hit regardless of the value? Not sure that's what we want.

Reply:
This is right. I have added option to check default values now.